package.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "service-communication-wrapper",
+  "name": "@quizizz/service-communication",
   "version": "1.0.0",
```

Comment:
we should update the version to reflect the release we'll draft from this branch, after merging the PR

Reply:
Ideally every release has it's own branch, in this case it's just fixit/main